systemd/tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
```

Comment:
I bet you could communicate to dbus from outside of a docker container, would be cool to try to write an integration test for this

Reply:
I gave it a try but I didn't manage to find a solution for this unfortunately. Used a Docker image with `systemd` in it, but couldn't get the connection to `D-Bus` to work.

Reply:
What container image did you use?

Reply:
This one: https://hub.docker.com/r/centos/systemd and I also tried a custom one based on that image but still couldn't get to the bottom of whether it was possible to communicate to dbus.
I'm currently trying to mock instead.